packages/cli/src/controllers/passwordReset.controller.ts

```diff
@@ -46,7 +47,14 @@ export class PasswordResetController {
 	/**
 	 * Send a password reset email.
 	 */
-	@Post('/forgot-password')
+	@Post('/forgot-password', {
+		middlewares: [
+			rateLimit({
+				windowMs: 15 * 60 * 1000, // 15 minutes
+				limit: 10, // Limit each IP to 10 requests per `window` (here, per 15 minutes).
```

Comment:
Can we change these to 
```suggestion
				windowMs: 5 * 60 * 1000, // 5 minutes
				limit: 5, // Limit each IP to 5 requests per `window` (here, per 5 minutes).
```